test/performance/performance/indexers/elastic.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2023 The go-commons Authors.
```

Comment:
Copied from github.com/cloud-bulldozer/go-commons and not added as a dependency because it brings in irrelevant deps with require replace directives (too noisy).